test/page-detect.js

```diff
@@ -0,0 +1,208 @@
+	]);
+
+	urlsDontMatch(t, pageDetect.isPRFiles, [
+		'https://github.com/sindresorhus/refined-github/pull/148',
+		'https://github.com/sindresorhus/refined-github/pull/commits',
+		'https://github.com/sindresorhus/refined-github/pulls'
+	]);
+});
+
+test('isPRCommit', t => {
+	urlsMatch(t, pageDetect.isPRCommit, [
+		'https://github.com/sindresorhus/refined-github/pull/148/commits/0019603b83bd97c2f7ef240969f49e6126c5ec85',
+		'https://github.com/sindresorhus/refined-github/pull/148/commits/00196'
+	]);
+
+	urlsDontMatch(t, pageDetect.isPRCommit, [
+		'https://github.com/sindresorhus/refined-github/pull/148',
+		'https://github.com/sindresorhus/refined-github/pull/commits',
+		'https://github.com/sindresorhus/refined-github/pulls'
+	]);
+});
+
+test('isCommitList', t => {
+	urlsMatch(t, pageDetect.isCommitList, [
+		'https://github.com/sindresorhus/refined-github/commits/master?page=2',
+		'https://github.com/sindresorhus/refined-github/commits/test-branch',
+		'https://github.com/sindresorhus/refined-github/commits/0.13.0',
+		'https://github.com/sindresorhus/refined-github/commits/230c2',
+		'https://github.com/sindresorhus/refined-github/commits/230c2935fc5aea9a681174ddbeba6255ca040d63'
+	]);
+
+	urlsDontMatch(t, pageDetect.isCommitList, [
+		'https://github.com/sindresorhus/refined-github/pull/148',
+		'https://github.com/sindresorhus/refined-github/pull/commits',
+		'https://github.com/sindresorhus/refined-github/branches'
+	]);
+});
+
+test('isSingleCommit', t => {
+	urlsMatch(t, pageDetect.isSingleCommit, [
+		'https://github.com/sindresorhus/refined-github/commit/5b614b9035f2035b839f48b4db7bd5c3298d526f',
+		'https://github.com/sindresorhus/refined-github/commit/5b614'
+	]);
+
+	urlsDontMatch(t, pageDetect.isSingleCommit, [
+		'https://github.com/sindresorhus/refined-github/pull/148/commits',
+		'https://github.com/sindresorhus/refined-github/branches'
+	]);
+});
+
+test('isReleases', t => {
+	urlsMatch(t, pageDetect.isReleases, [
+		'https://github.com/sindresorhus/refined-github/releases'
+	]);
+
+	urlsDontMatch(t, pageDetect.isReleases, [
+		'https://github.com/sindresorhus/refined-github',
+		'https://github.com/sindresorhus/refined-github/graphs'
+	]);
+});
+
+test('isBlame', t => {
+	urlsMatch(t, pageDetect.isBlame, [
+		'https://github.com/sindresorhus/refined-github/blame/master/package.json'
+	]);
+
+	urlsDontMatch(t, pageDetect.isBlame, [
+		'https://github.com/sindresorhus/refined-github/blob/master/package.json'
+	]);
+});
+
+test('getOwnerAndRepo', t => {
+	const ownerAndRepo = {
+		'https://github.com/sindresorhus/refined-github/pull/148': {
+			ownerName: 'sindresorhus',
+			repoName: 'refined-github'
+		},
+		'https://github.com/DrewML/GifHub/blob/master/.gitignore': {
+			ownerName: 'DrewML',
+			repoName: 'GifHub'
+		}
+	};
+
+	Object.keys(ownerAndRepo).forEach(url => {
+		location.href = url;
+		t.deepEqual(ownerAndRepo[url], pageDetect.getOwnerAndRepo());
```

Comment:
Note: I updated ava to `*` in `package.json` in a recent commit - if `t.deepEqual` fails, run npm install to update to the latest version